src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    public Figure getRandomFigure() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Random random = new Random();
+        int number = random.nextInt(5);
```

Comment:
Avoid using magic numbers directly in your code. According to the checklist, all magic numbers should be constants. Consider defining a constant for the number of figure types.